web/src/app/import-export/export/export.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { fileURLToPath } from 'url';
+import { ImportExportService } from '../shared/import-export.service';
+
+@Component({
+  selector: 'app-export',
+  templateUrl: './export.component.html',
+  styleUrls: ['./export.component.scss']
+})
+export class ExportComponent implements OnInit {
+  @Input() treeId: string;
+
+  constructor(
+    private service: ImportExportService
+  ) { }
+
+  ngOnInit(): void {
+  }
+
+  downloadFile() {
+    this.service.getTreeFileExport(this.treeId).subscribe(result => {
+      var blob = new Blob([result], { type: 'text/xml' });
```

Comment:
Can you add mapping from file extension? There it is not that important, but probably when adding persons files you should use some library that maps file extension to mime type, forgot to check that in "Add persons file" Merge Request

Reply:
It is export not import, but now I changed it and I read type of file from response.